apps/modernization-ui/src/pages/patient/profile/PatientProfile.tsx

```diff
@@ -84,8 +84,8 @@ export const PatientProfile = () => {
     return (
         <ProfileProvider id={id}>
             <div className="height-full main-banner">
-                <div className="bg-white grid-row flex-align-center flex-justify border-bottom-style">
-                    <h1 className="font-sans-xl text-medium">Patient profile</h1>
+                <div className="padding-left-2 padding-right-1 bg-white grid-row flex-align-center flex-justify border-bottom-style">
+                    <h1 className="font-sans-xl text-medium">Patient Profile</h1>
```

Comment:
issue: The page title should remain sentence cased.  "Patient Profile"

Reply:
Done